src/utils/shared/sidebar.js

```diff
@@ -21,14 +21,14 @@
 const startCase = require('lodash/startCase')
 const sidebar = require('../../../content/docs/sidebar.json')
 
-const PATH_ROOT = '/doc/'
+const PATH_ROOT = '/doc'
 const FILE_ROOT = '/docs/'
 const FILE_EXTENSION = '.md'
```

Comment:
I'm getting a 404 in /doc/home, probably because of the changes to this file. I'm confused though because this changes come from a previous PR and seem to have been reverted in master, which I merged in but didn't revert all of this... I'll try reverting manually... ⏳

Reply:
Ok that worked. Pushed.